jena-db/jena-tdb2/src/main/java/org/apache/jena/tdb2/xloader/ProcIndexBuildX.java

```diff
@@ -143,31 +143,49 @@ private static long sort_build_index(Logger LOG, String datafile, DatasetGraph d
         Process proc2;
         OutputStream toSortOutputStream; // Not used. Input is a file.
         InputStream fromSortInputStream;
+
         try {
             //LOG.info("Step : external sort : "+indexName);
             //if ( sortArgs != null ) {}
+
+
             List<String> sortCmdBasics = Arrays.asList(
                  "sort",
                     "--temporary-directory="+TMPDIR, "--buffer-size=50%",
                     "--parallel=2", "--unique"
```

Comment:
Default `sort` on OS X doesn't have the `--parallel` option, should that be a configurable feature of the loader?

(I know OS X systems are most likely to have SSDs and not need this loader so much)

Reply:
There is enough work at the moment just to get it stable and effective on Linux!

While this tool is not too demanding on the machine, I can type and do email while it is running, are we really going to see usage on a Mac Book? It takes many hours to run so even from that POV, it's an offline/server side tool.

I'm inclined to do the variety refinement when it's working properly.  `jq` does need installing so a check in the shell script.

Reply:
No worries, maybe just something to mention in the documentation that a `sort` with `--parallel` support is assumed

Reply:
Maybe we can just check the OS and warn if used in Mac OS, and add notes to the docs that this is a Linux tool. I think the `find` command in Mac OS is also different (remember being given a Mac OS and returning after I couldn't find anything :laughing: ).